client/app/src/main/java/com/microsoft/research/karya/ui/base/BaseActivity.kt

```diff
@@ -36,10 +36,12 @@ import java.io.File
 import java.text.SimpleDateFormat
 import java.util.*
 
-abstract class BaseActivity(
-    private val useAssistant: Boolean = false,
-    private val playAssistantOnResume: Boolean = true
-) : AppCompatActivity() {
+abstract class BaseActivity() : AppCompatActivity() {
+
+    private val useAssistant: Boolean = false
+    private val playAssistantOnResume: Boolean = false
+    //TODO: Removing from the constructor since hilt doesnt support this.
+    // We can manipulate this in other ways
```

Comment:
Removing it is not required since we are not planning to use it in the new activities.

Reply:
In our current implementation the activities extends BaseActivity. There was some initialization in constructor which I removed for now since hilt contains a constructor with default parameters. this is currently not supported by the Gradle plugin. This is only done for now to support current implementation and we would not require it at all after BaseActivity is removed